.github/workflows/static-analyzers.yml

```diff
@@ -0,0 +1,15 @@
+name: Static Analyzers
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  cpplint:
+    runs-on: ubuntu-latest
+    container: sharaku/cpplint:latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run cpplint
+      run: cpplint --counting=detailed --recursive examples benchmarks src tests
```

Comment:
you thinking a clang-tidy run would be in this same file eventually?

Reply:
I'm thinking analyzers that don't take long to run (<2 minutes).